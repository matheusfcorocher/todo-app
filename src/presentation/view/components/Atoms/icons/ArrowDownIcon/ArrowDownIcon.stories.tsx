import React from 'react';
import { ComponentStory, ComponentMeta } from '@storybook/react';
import ListIcon from './ArrowDownIcon';

export default {
  title: 'Atoms/ListIcon',
  component: ListIcon,
  decorators: [
    (Story) => (
      <ul style={{ listStyleType: 'none', margin: 0, padding: 0 }}>
        <Story />
      </ul>
    ),
  ]
} as ComponentMeta<typeof ListIcon>;

const Template: ComponentStory<typeof ListIcon> = (args) => <ListIcon />;

export const Default = Template.bind({});
Default.args = {};

Default.args = {
};