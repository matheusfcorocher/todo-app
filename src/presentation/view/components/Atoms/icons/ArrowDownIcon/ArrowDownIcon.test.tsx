import React, { useState } from 'react';
import { render, screen, cleanup } from '@testing-library/react';
import ArrowDownIcon from './ArrowDownIcon';

describe("::Components ::Atoms ::ArrowDownIcon", () => {
    test('renders icon of ArrowDownIcon', () => {
        render(<ArrowDownIcon />);
        const icon = screen.getByRole("image");
        expect(icon).toMatchSnapshot();
    })
})
