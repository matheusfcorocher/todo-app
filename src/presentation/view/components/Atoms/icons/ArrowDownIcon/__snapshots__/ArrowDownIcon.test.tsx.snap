// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`::Components ::Atoms ::ArrowDownIcon renders icon of ArrowDownIcon 1`] = `
<svg
  class="undefined"
  preserveAspectRatio="none"
  role="image"
  viewBox="0 0 24 24"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M0 0h24v24H0V0z"
    fill="none"
  />
  <path
    d="M7.41 8.59L12 13.17l4.59-4.58L18 10l-6 6-6-6 1.41-1.41z"
    fill=""
  />
</svg>
`;
